CHANGELOG.md

```diff
@@ -220,9 +220,9 @@ All notable changes to this project will be documented in this file.
 ### Added
 * `aswfdocker packages` lists all ci packages.
 * `aswfdocker images` lists all ci images.
-* `aswfdocker migrate` allows migration of docker packages from one organisation to another.
+* `aswfdocker migrate` allows migration of docker packages from one organization to another.
```

Comment:
Arguably both organization and organisation is OK in UK English, but only organization is OK in US English, so the zeds have it (right?).

Reply:
My rationale is that the ASWF as a US legal body should use US spelling, but it's only a suggestion after all.